<launch>

  <param name="/Brain5Camera/vrpn_server_ip" value="192.168.1.103"/>
  <!-- ros_vrpn_client -->
  <node pkg="ros_vrpn_client" type="ros_vrpn_client" name="$(optenv MYSELF Brain5Camera)" required="true"/>

  <!-- irobot_waypoint_brain -->
  <node pkg="irobot_waypoint_brain" type="irobot_waypoint_brain" name="irobot_waypoint_brain" required="true" output="screen"/>

  <!-- pgrcamera_driver -->
  <param name="/basic_camera_driver/left_cam_url" value="file://$(find pgrcamera_driver)/include/left_calib_yan.yaml" /> 
  <param name="/basic_camera_driver/right_cam_url" value="file://$(find pgrcamera_driver)/include/right_calib_yan.yaml"/> 
  <node pkg="pgrcamera_driver" type="basic_camera_driver" name="basic_camera_driver" required="true"/>

  <!--stereo_image_proc-->  
  <node pkg="stereo_image_proc" type="stereo_image_proc" name="stereo_image_proc" ns="stereo" required="true">
        <param name="approximate_sync" type="bool" value="true"/>
  </node>

</launch>
